Content.Server/Light/EntitySystems/PoweredLightSystem.cs

```diff
@@ -244,13 +244,13 @@ public void TryDestroyBulb(EntityUid uid, PoweredLightComponent? light = null)
             }
             else
             {
-                powerReceiver.Load = (light.On && lightBulb.State == LightBulbState.Normal) ? lightBulb.PowerUse : 0;
 
                 switch (lightBulb.State)
                 {
                     case LightBulbState.Normal:
                         if (powerReceiver.Powered && light.On)
                         {
+                            powerReceiver.Load = lightBulb.PowerUse;
```

Comment:
So because this only sets the load to a non-zero value if the device is currently powered, whenever the device looses power the load gets set to 0 instead, which then means it then is considered powered. This causes the light to rapidly switch between on and off.

The easiest fix is probably to just move that original line to be after the switch block.

Reply:
Done! After this PR passes and I get some time, I'm reading and understanding everything about power, even if it takes me months.